gslib/commands/cat.py

```diff
@@ -128,6 +128,10 @@ def RunCommand(self):
           show_header = True
         elif o == '-r':
           request_range = a.strip()
+          # This if statement ensures the full object is returned
+          # instead of throwing a CommandException.
+          if request_range == '-':
+            break
```

Comment:
Did you mean `continue` instead of `break` here? What happens if both the -h and -r flags are used?

Reply:
yup, you are right. nice find!